mkdocs.yml

```diff
@@ -11,12 +11,15 @@ plugins:
   - mkdocs-jupyter:
       include_source: True
       theme: dark
+  - blog:
+      blog_dir: ./articles
+      authors_file: "{blog}/.authors.yml"
 nav:
   - Home: index.md
   - What is Data Science?:
       - ... | what_is_data_science/*.md
   - Articles:
-      - ... | articles/*.md
+      - ... | articles/**.md
```

Comment:
just for my own understanding - how come it needs two *s?

Reply:
My own mistake - seems redundant. For some reason I though that was needed to make it recursive... but seems not!